.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,33 @@
+<!--
+
+    Sonatype Nexus (TM) Open Source Version
+    Copyright (c) 2017-present Sonatype, Inc.
+    All rights reserved. Includes the third-party code listed at http://links.sonatype.com/products/nexus/oss/attributions.
+
+    This program and the accompanying materials are made available under the terms of the Eclipse Public License Version 1.0,
+    which accompanies this distribution and is available at http://www.eclipse.org/legal/epl-v10.html.
+
+    Sonatype Nexus (TM) Professional Version is available from Sonatype, Inc. "Sonatype" and "Sonatype Nexus" are trademarks
+    of Sonatype, Inc. Apache Maven is a trademark of the Apache Software Foundation. M2eclipse is a trademark of the
+    Eclipse Foundation. All other trademarks are the property of their respective owners.
+
+-->
+## How to be a contributor to this project
+
+### Are you submitting a pull request?
+
+* Fill out a CLA for us, so we can sort out all the legal parts of contributing. You can get all the information for
+this [here](https://books.sonatype.com/nexus-book/reference/contrib.html). Follow the CLA process and email in your
+form. We are working on a way to make this simpler, as well.
```

Comment:
Minor: Maybe a nod that it's the same for dev and doc since that goes to a doc page, might be confusing?